blocks/magazine-hero/magazine-hero.js

```diff
@@ -16,6 +16,9 @@ export default async function decorate(block) {
 
   const overlay = document.createElement('div');
   overlay.classList.add('text-overlay');
+  if (config.titlecolor) {
```

Comment:
let's read the actual title color, make sure it's `dark`. 
Also try to use `Title color` instead of `TitleColor` in the doc.